x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/operator/EvalOperator.java

```diff
@@ -25,6 +27,7 @@ public Operator get(DriverContext driverContext) {
 
         @Override
         public String describe() {
+            // TODO ThrowingDriverContext blows up when combined with Concat
             return "EvalOperator[evaluator=" + evaluator.get(new ThrowingDriverContext()) + "]";
```

Comment:
I think we should generate `toString` implementations for the `evaluator` instances. We are generating these classes - I think it's just making the `toString`.

Reply:
Yeah, generation would be great. I know that you raised this comment before also. I'm not sure why this blows up with Concat. Does Concat::toString need to circuit break?